server/ServerApp.cpp

```diff
@@ -379,6 +379,14 @@ void ServerApp::AsyncIOTimedoutHandler(const boost::system::error_code& error) {
     }
 }
 
+void ServerApp::UpdateEmpireTurnReceive(int empire_id, int turn, bool success) {
```

Comment:
Maybe should be called turn received not just receive? And possibly sent instead of received?

Reply:
I guess it updates only when the send rwturns sucess so "received" should be OK

Reply:
Changed.